src/internal/storage/fileset/metadata_store.go

```diff
@@ -2,21 +2,21 @@ package fileset
 
 import (
 	"context"
+	"fmt"
 	"testing"
 
 	"github.com/jmoiron/sqlx"
 	"github.com/pachyderm/pachyderm/v2/src/internal/dbutil"
-	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
 	"github.com/pachyderm/pachyderm/v2/src/internal/require"
 )
 
 var (
 	// ErrFileSetExists path already exists
-	ErrFileSetExists = errors.Errorf("path already exists")
+	ErrFileSetExists = fmt.Errorf("path already exists")
```

Comment:
This prevents us from checking for theses errors with `==`.  It's generally better to have each error be a separate type so that the instances can carry stack information, but we aren't ready to do that yet.